tests/unit/s2n_shutdown_test.c

```diff
@@ -395,20 +406,172 @@ int main(int argc, char **argv)
         EXPECT_FAILURE_WITH_ERRNO(
                 s2n_recv(conn, recv_buffer, sizeof(recv_buffer), &blocked),
                 S2N_ERR_BAD_MESSAGE);
-        EXPECT_TRUE(s2n_stuffer_space_remaining(&conn->header_in) < sizeof(header_bytes));
 
         /* Clear the blinding delay so that we can call s2n_shutdown */
         EXPECT_TRUE(conn->delay > 0);
-        conn->delay = 0;
+        conn->write_timer.time = 0;
```

Comment:
Like the comment says-- to clear the blinding delay. I'm just now clearing it by setting the timer time to waaaaaay in the past instead of setting delay to zero.

Reply:
Ahhh got it ok, thanks!